packages/templates/src/getMetaData.ts

```diff
@@ -17,13 +17,18 @@
  * Internal dependencies
  */
 import { TEMPLATE_NAMES } from './constants';
+import type { MetaData } from './types';
 
-export function getTemplateMetaData() {
+type Data = {
+  default: MetaData;
+};
```

Comment:
Why define this here?

Reply:
Because it's very specific to this one function here only, in this case, adding it next to the function has been a common practice for readability across the project so far.